B2MML-ProcessEvents.xsd

```diff
@@ -86,28 +87,28 @@
   
 <xsd:complexType name = "MaterialTestSpecificationType">
     <xsd:sequence>
-      <xsd:element   name  = "TestSpecificationType"/>
-      <xsd:element   name = "MaterialTestSpecification"  type = "etp:MaterialTestSpecificationType"  minOccurs = "0" maxOccurs = "unbounded"/>   
+      <xsd:element name = "TestSpecification" type = "TestSpecificationType"/>
+      <xsd:element name = "MaterialTestSpecification"  type = "etp:MaterialTestSpecificationType"  minOccurs = "0" maxOccurs = "unbounded"/>   
     </xsd:sequence>
   </xsd:complexType>
 
  <xsd:complexType name = "PhysicalAssetCapabilityTestSpecificationType">
     <xsd:sequence>
-      <xsd:element   name  = "TestSpecificationType"/>
+      <xsd:element name = "TestSpecification" type = "TestSpecificationType"/>
       <xsd:element name = "PhysicalAssetCapabilityTestSpecification"  type = "etp:PhysicalAssetCapabilityTestSpecificationType" minOccurs = "0" maxOccurs = "unbounded"/>
     </xsd:sequence>
   </xsd:complexType> 
   
    <xsd:complexType name = "EquipmentCapabilityTestSpecificationType">
     <xsd:sequence>
-      <xsd:element   name  = "TestSpecificationType"/>
+      <xsd:element name = "TestSpecification" type = "TestSpecificationType"/>
       <xsd:element name = "EquipmentCapabilityTestSpecification"  type = "etp:EquipmentCapabilityTestSpecificationType"  minOccurs = "0" maxOccurs = "unbounded"/>
     </xsd:sequence>   
   </xsd:complexType>
   
   <xsd:complexType name = "QualificationTestSpecificationType">
           <xsd:sequence>
-      	  <xsd:element   name  = "TestSpecificationType"/>
+      	  <xsd:element name = "TestSpecification" type = "TestSpecificationType"/>

```

Comment:
This fix is for all the "qualification test specified/adjusted/retired event' templates at the end of the ProcessEvents.xsd.
Issue still to be raised.